Assets/Scripts/Calendar UI.cs

```diff
@@ -54,9 +69,15 @@ private void OnDestroy()
     {
         EventManager.Instance.onForwardClick -= HandleForwardClick;
         EventManager.Instance.onBackwardClick -= HandleBackwardClick;
-    }
+        EventManager.Instance.onCellSelect -= HandleSelectCell;
+        EventManager.Instance.onDESelectAllCells -= HandleDESelectAllCells;
+        EventManager.Instance.onCellImageSelect -= HandleCellImageSelect;
 
+    }
 
+    // _____________________________________________________________________________________
+    //   EVENT - DEPENDANT FUNCTIONS
+    // _____________________________________________________________________________________
```

Comment:
Really like these kinds of "decorative" comments - comments should be used to decorate code (as long as it doesnt impact readability), or explain WHY a bit of code is working the way it does. 

I see a lot of comments like this

```
// get the RectTransform component of the child
rectTrans = child.GetComponent<RectTransform();

//set it's anchors to the parent's anchors
rectTrans.anchors = parentTrans.anchors; 
```

in peoples code and this is totally unhelpful. Your variable and function names should be responsible for indicating the intent of your code. Just a rant you can resolve this comment

Reply:
=P